src/infinity-menu.js

```diff
@@ -113,12 +115,16 @@ export default class InfinityMenu extends React.Component {
 				return this.findFilted(p, c, k);
 			}, []) : [];
 			if (filteredSubFolder.length !== 0) {
-				node.isOpen = true;
+				node.isSearchOpen = true;
```

Comment:
to make search working correct, we need to have an searchOpen property to mark what is open inside search and not affect the original open status